### PR TITLE
Add multi-sig game verifier in child chain

### PR DESCRIPTION
**Pull request by syuhei176**
## Description

Add simple multi-sig game verifier in Child Chain.

### CODE REVIEW

File: packages/chamber-childchain/lib/verifier/index.js [hunk 1]
```diff
@@ -1,16 +1,30 @@
 const std = require('./std');
+const MultisigGame = require('./MultisigGame');
+
+/**
+ * verifiers should be loaded from config file
+ */
+const verifiers = {
+  "0x9fbda871d559710256a2502a2517b794b482db40": MultisigGame
```
Comment (syuhei176) [inline packages/chamber-childchain/lib/verifier/index.js]: This mapping should be a config file.

Comment (shogochiai) [reply packages/chamber-childchain/lib/verifier/index.js]: `dotenv`?

Comment (syuhei176) [reply packages/chamber-childchain/lib/verifier/index.js]: This is the mapping of verifier address.
define as JSON-string in env?
or we can use one address per a plasma?
```
const verifiers = {
  "0x9fbda871d559710256a2502a2517b794b482db40": MultisigGame,
  "0x111111": Contract1,
  "0x111112": Contract2
...
}
```

Comment (syuhei176) [reply packages/chamber-childchain/lib/verifier/index.js]: hard coding is OK in the first implementation.
We need to generate the whole `chamber-childchain` project from chamber-lang generator.
@shogochiai

**Pull request closed without merging**